indigo/distribution.yaml

```diff
@@ -12877,6 +12877,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_moveit.git
       version: master
     status: maintained
+  rqt_mrta:
+    doc:
+      type: git
+      url: https://github.com/adrianohrl/rqt_mrta.git
+      version: 0.2.5
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/rqt_mrta-release.git
+      version: 0.2.5-0
+    source:
+      test_pull_requests: true
```

Comment:
To check did you grant ros-pull-request-builder access as outlined here: http://wiki.ros.org/buildfarm/Pull%20request%20testing

Reply:
Awaiting ros-pull-request-builder's response

Reply:
I see it has access to [alliance](https://github.com/adrianohrl/alliance), but not rqt_mrta

Reply:
> Awaiting ros-pull-request-builder's response

Confimed :+1:

Reply:
all setted